bucket/gitea.json

```diff
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://gitea.io/",
+    "version": "1.9",
+    "description": "A painless self-hosted Git service.",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://dl.gitea.io/gitea/1.9/gitea-1.9-windows-4.0-amd64.exe#/gitea.exe",
+            "hash": "a38be091b407b7ade13fa637ea71588ca58c5afebd222c95fd1ba27630e08c47"
+        },
+        "32bit": {
+            "url": "https://dl.gitea.io/gitea/1.9/gitea-1.9-windows-4.0-386.exe#/gitea.exe",
+            "hash": "81ff3c007674c41d77653d6d0f5a2da4aa09d42f2bb818f4ab3c49d3ac16481d"
+        }
+    },
+    "post_install": [
+        "echo 'Creating $persist_dir\\custom\\conf.app.ini'",
+        "if(!(test-path \"$persist_dir\\custom\\conf\")) { mkdir \"$persist_dir\\custom\\conf\" > $null }",
+        "if(!(test-path \"$persist_dir\\custom\\conf\\app.ini\")) {",
+        "    $gitea = \"[database]`nPATH = $persist_dir/data/gitea.db\"",
+        "    $gitea += \"`n[repository]`nROOT = $persist_dir/repositories\"",
+        "    $gitea += \"`n[log]`nROOT_PATH = $persist_dir/log\"",
+        "    echo $gitea | % {$_.replace(\"\\\",\"/\") } | out-file -encoding oem \"$persist_dir\\custom\\conf\\app.ini\"",
```

Comment:
- Fix encoding
- Replace could be called on array without foreach
- Simple `Set-Content ($gitea -replace) ..app.ini -Encoding ASCII` is good